UniversalAlgebra/GaloisConnections/Properties.lagda

```diff
@@ -44,10 +44,10 @@ module _ {α ρ : Level}{𝒜 : Type α} where
  tran ≐-iseqv (u₁ , u₂) (v₁ , v₂) = v₁ ∘ u₁ , u₂ ∘ v₂
 
 
-module _ {α ρ : Level}{𝒜 : Type α} where
+module _ {α : Level} (ρ : Level) (𝒜 : Type α) where
 
- PosetOfSubsets : Poset (lsuc (α ⊔ ρ)) (α ⊔ ρ) (α ⊔ ρ)
- Carrier PosetOfSubsets = Pred 𝒜 (α ⊔ ρ)
+ PosetOfSubsets : Poset (α ⊔ lsuc ρ) (α ⊔ ρ) (α ⊔ ρ)
+ Carrier PosetOfSubsets = Pred 𝒜 ρ
```

Comment:
Besides improving the levels, I expected you would suggest a better approach to PosetOfSubsets.   It's such a "canonical" example of poset; I am (would be) surprised (if) it's not already implemented in a nicer way somewhere.

Reply:
I do try to do as few changes per PR as possible. Also, I'm not so familiar with the `Poset` area of stdlib. And, for that matter, posets themselves! They were kind of skipped in my mathematical education, so my knowledge of them is thin.

Reply:
After looking around, it seems that this particular poset is indeed not defined in stdlib. I've submitted issues to see where the pieces should go.